modules/distr-tracing-tempo-update-olm.adoc

```diff
@@ -0,0 +1,9 @@
+//Module included in the following assemblies:
+//
+//* distr_tracing_install/dist-tracing-tempo-updating.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-update-olm_{context}"]
+= Updating the {TempoShortName} by using the OLM
+
+For version upgrades, the {TempoOperator} uses the Operator Lifecycle Manager (OLM), which controls installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. The OLM runs in {product-title} by default. The OLM queries for available Operators as well as upgrades for installed Operators. When the {TempoOperator} is upgraded to the new version, it scans for running TempoStack instances that it manages and upgrades them to the version corresponding to the Operator's new version.
```

Comment:
Consider breaking this paragraph up into sections (intro, OLM info, Tempo-specific info).

```suggestion
For version upgrades, the {TempoOperator} uses the Operator Lifecycle Manager (OLM), which controls installation, upgrade, and role-based access control (RBAC) of Operators in a cluster.

The OLM runs in {product-title} by default. The OLM queries for available Operators as well as upgrades for installed Operators. 

When the {TempoOperator} is upgraded to the new version, it scans for running TempoStack instances that it manages and upgrades them to the version corresponding to the Operator's new version.
```

Reply:
Thank you so much. Applying this manually.